pyproject.toml

```diff
@@ -39,7 +39,7 @@ dependencies = ["ase", "joblib", "phonopy", "pymatgen"]
 version = "0.0.4"
```

Comment:
let's also set a down-pin to the latest `ase`

Reply:
Will do

Reply:
Pinned to "ase>=3.23.0"

Reply:
maybe you forgot to push?